test/dune

```diff
@@ -2,7 +2,38 @@
 
 (test
  (name haz3ltest)
- (libraries haz3lcore alcotest junit junit_alcotest)
+ (libraries
+  haz3lcore
+  alcotest
+  sexplib
+  base
+  ptmap
+  incr_dom
+  haz3lmenhir
+  junit
+  junit_alcotest)
  (modes js)
+ (js_of_ocaml
+  (flags
+   (:include js-of-ocaml-flags-%{profile})))
  (preprocess
-  (pps js_of_ocaml-ppx)))
+  (pps
+   js_of_ocaml-ppx
+   ppx_let
+   ppx_sexp_conv
+   ppx_deriving.show
+   ppx_yojson_conv)))
+
+(env
+ (dev
+  (js_of_ocaml
+   (flags (:standard))))
+ (release
+  (js_of_ocaml
+   (flags (:standard)))))
+
+(rule
+ (write-file js-of-ocaml-flags-dev "(:standard --debuginfo --noinline)"))
+
+(rule
+ (write-file js-of-ocaml-flags-release "(:standard)"))
```

Comment:
We should update this to match the format in the other dune files or just revert to what's on dev